package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "wahtson",
-  "version": "1.0.4",
+  "version": "1.0.5",
```

```suggestion
  "version": "1.0.4",
```
Don't bump the version please - let me do that so the release GitHub action triggers :)